app/controllers/content_items_controller.rb

```diff
@@ -26,8 +26,6 @@ def present(content_item)
     presenter_name = content_item['schema_name'].classify + 'Presenter'
     presenter_class = Object.const_get(presenter_name)
     presenter_class.new(content_item)
-  rescue NameError
```

Comment:
i can put it in a seperate commit, but you think it needs a whole seperate PR, and deploy for something like that, that only effects development?